EjerciciosT15/src/paquetetres/LecturaArchivo.java

```diff
@@ -40,10 +40,12 @@ public static void leerRegistros() {
                 numeroEstablecimientosEntero = Integer.parseInt(numeroEstablecimientos);
                 
                 // agregar código aquí
-                
-                cadenaFinal = String.format("Cantón %s (%d)\n", 
+                if(numeroEstablecimientosEntero <=25){
+                cadenaFinal = String.format("%sCantón %s (%d)\n", 
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.